tests/BindingTests/Xamarin.Android.BindingRuntime-Tests/Tests/BindingTests.cs

```diff
@@ -0,0 +1,398 @@
+using System;
+using System.Collections.Generic;
+
+using NUnit.Framework;
+
+using Android.Graphics;
+using Android.Runtime;
+
+using Com.Xamarin.Android;
+
+namespace Xamarin.Android.BindingRuntime_Tests {
+
+	[TestFixture]
+	public class BindingTests {
+
+		[Test]
+		public void TestTimingCreateTimingIsCorrectType ()
+		{
+			var t = Com.Xamarin.Android.Timing.CreateTiming ();
+			Assert.IsTrue (t is Com.Xamarin.Android.Timing);
+		}
+
+#if TODO_7794
+		[Test]
+		public void TestResourceId ()
+		{
+			Assert.AreEqual (Resource.Id.action_settings, Com.Example.Javalib.MainActivity.ActionSettings);
+		}
+
+		[Test]
+		public void TestNativeLibDllImport ()
+		{
+			Assert.AreEqual (TestNativeLib.Binding.SampleFunction (), 0xf00);
+		}
+
+		[Test]
+		public void TestNativeLibDllImportInEmbeddedArchive ()
+		{
+			Assert.AreEqual (TestNativeLib.Binding.SampleFunction2 (), 0xf200);
+		}
+#endif  // TODO_7794
+		
+		[Test]
+		public void NamespaceTransforms ()
+		{
+			// Really the only test here is that the type exists in the binding.
+			// If the transforms were not working it would be 'Com.Xamarin.Example.NamespaceTransform'.
+			var t = new Transformed.Namespace.NamespaceTransform ();
+			Assert.IsNotNull (t);
+		}
+
+		[Test]
+		public void TestBxc4288 ()
+		{
+			var t = new Com.Xamarin.Android.Bxc4288 ();
+			var c = t.UseColors (Color.Gray);
+			Assert.AreEqual (Color.Gray, c);
+		}
+
+		[Test]
+		public void Hello ()
+		{
+			using (var v = new HélloÊncodingIssues ()) {
+			}
+		}
+
+		[Test]
+		public void EnsureTypesAreBound ()
+		{
+#pragma warning disable 0219
+			Com.Xamarin.Android.Bxc9446 ignore  = null;
+#pragma warning restore 0219
+		}
+
+		//[Test]
+		//public void Arrays ()
```

Comment:
@jpobst: why is *this* test disabled?  `Com.Xamarin.Android.DataHandler` is in `DataHandler.java`, which is added/copied as part of this PR.

Reply:
It was disabled because Marshal Methods broke it:
https://github.com/xamarin/xamarin-android/issues/7693

As that has now been fixed this test should be enabled.